CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ All notable changes to this crate are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.7.0] - 2024-10-24
+
+### Added
+
+- Allow decoding with a memory limit. ([616](https://github.com/paritytech/parity-scale-codec/pull/616))
```

Comment:
- Introduce `CountedInput`, an wrapper on `Input` that counts the byte read. ([630](https://github.com/paritytech/parity-scale-codec/pull/630))